client/src/components/ChatTile.jsx

```diff
@@ -66,3 +61,1913 @@ function ChatTile (props) {
 }
 
 export default ChatTile;
+
+// provided by twitch
```

Comment:
Is there a documentation link you could provide in comments?

Reply:
good idea, fixed